tests/TestBtreePrinter.cpp

```diff
@@ -0,0 +1,35 @@
+#include <internal/storage/btree/Btree.h>
+#include <internal/storage/btree/BtreePrinter.h>
+
+#include <iostream>
+#include <ranges>
+using namespace std::ranges::views;
+
+using namespace internal::btree;
+using namespace internal::btree::util;
+
+using config_ = DefaultBtreeConfig;
+using bt_t = Btree<config_>;
+using btprinter_t = BtreeYAMLPrinter<config_>;
+using node_t = bt_t::Node;
+using it_t = bt_t::Iterator;
+
+// Btree Print Single Layer
+int main() {
```

Comment:
TestBtreePrinter - what he does and where he has to do it? By cause of, when I execute it - nothing happen.

Reply:
I am unsure whether I understand you correctly, but `TestBtreePrinter.cpp` actually outputs a file called `bpt.yml` which is the visual representation of the tree. It should reside in the `build/` directory. Are you sure that there is no such file created?